src/task_1.2/task2.js

```diff
@@ -0,0 +1,28 @@
+const csv = require('csvtojson');
+const { pipeline } = require('stream/promises');
+const path = require('path');
+const fs = require('fs');
+const fsPromises = require('fs').promises;
+
+const readableStream = fs.createReadStream(path.join(__dirname, 'csv/nodejs-hw1-ex1.csv'));
```

Comment:
Consider storing path to the files as constants at top of the file (after imports).